Example code in XPT2046 touch documentation not working due to spurious comma

<!-- Thanks for reporting a bug for this project. READ THIS FIRST:

- Provide as many details as possible. Simply saying "X gives bug" or "X gives error" is not enough!
- Paste logs, configuration sample and code into the backticks (```).
- Read through the template carefully and fill out all missing details.
- Please also search for similar issues in this issue tracker first and read through the ESPHome FAQ.

DO NOT DELETE ANY TEXT from this template! Otherwise the issue may be closed without a comment.
-->

**Operating environment/Installation (Hass.io/Docker/pip/etc.):**
<!--
Please provide details about your environment below this line. -->
`pip 21.1.2 from /usr/local/lib/python3.9/dist-packages/pip (python 3.9)`

**ESP (ESP32/ESP8266, Board/Sonoff):**
<!--
Please provide details about which ESP you're using below.
-->
`esp32dev` (DevKitC V4)

**ESPHome version (latest production, beta, dev branch)**
<!--
ESPHome version like v1.14 or 1.15-dev
-->
1.19.1

**Affected component:**
<!--
Please add the link to the documentation at https://esphome.io/index.html of the component in question.
-->
[`xpt2046`](https://www.esphome.io/components/binary_sensor/xpt2046.html)

**Description of problem:**

The example code in the *Calibration* section of the documentation for the new XPT2046 touch integration does not compile due to a spurious comma at the end of the second-to-last line. That line should read,

```
              id(touchscreen).y_raw
```

*without* the comma at the end, otherwise compilation results in the following error:

```
Compiling .pioenvs/mydevice/src/main.cpp.o
In file included from src/esphome/components/api/api_server.h:6:0,
                 from src/esphome/components/api/api_connection.h:7,
                 from src/esphome.h:2,
                 from src/main.cpp:3:
mydevice.yaml: In lambda function:
src/esphome/core/log.h:101:99: error: expected primary-expression before ')' token
   esp_log_printf_(ESPHOME_LOG_LEVEL_INFO, tag, __LINE__, ESPHOME_LOG_FORMAT(format), ##__VA_ARGS__)
                                                                                                   ^
src/esphome/core/log.h:146:28: note: in expansion of macro 'esph_log_i'
 #define ESP_LOGI(tag, ...) esph_log_i(tag, __VA_ARGS__)
                            ^
mydevice.yaml:159:9: note: in expansion of macro 'ESP_LOGI'
*** [.pioenvs/mydevice/src/main.cpp.o] Error 1
================================================= [FAILED] Took 1.71 seconds =================================================
```

**Problem-relevant YAML-configuration entries:**
```yaml
# …
xpt2046:
  id: touchscreen
  cs_pin: 17
  irq_pin: 16
  dimension_x: 240
  dimension_y: 320
  on_state:
    - lambda: |-
        if (touched)
          ESP_LOGI("cal", "x=%d, y=%d, x_raw=%d, y_raw=%d",
              id(touchscreen).x,
              id(touchscreen).y,
              id(touchscreen).x_raw,
              id(touchscreen).y_raw,
              );
# …
```

**Logs (if applicable):**
<!--
Please copy the debug log here. If possible, also connect to the ESP over USB and copy those logs into the backticks.
-->

(Not applicable, fails at compile time.)

**Additional information and things you've tried:**

Removing the spurious comma solves the problem for me.

<!-- LEAVE THIS LINE AS-IS AND DON'T DELETE IT, OTHERWISE THE ISSUE WILL BE CLOSED AUTOMATICALLY. -->

--- a/components/binary_sensor/xpt2046.rst
+++ b/components/binary_sensor/xpt2046.rst
@@ -192,7 +192,7 @@ values nor ``swap_x_y``.
                   id(touchscreen).x,
                   id(touchscreen).y,
                   id(touchscreen).x_raw,
-                  id(touchscreen).y_raw,
+                  id(touchscreen).y_raw
                   );
 
 Get a stylus or a similar object, run the project and touch the corners of the screen at

